Modules/Microsoft365DSC/DSCResources/MSFT_EXORecipientPermission/MSFT_EXORecipientPermission.schema.mof

```diff
@@ -0,0 +1,16 @@
+[ClassVersion("1.0.0.0"), FriendlyName("EXORecipientPermission")]
+class MSFT_EXORecipientPermission : OMI_BaseResource
+{
+    [Key, Description("The mailbox the permission should be given on.")] String Identity;
+    [Key, Description("The account to give the permission to.")] String Trustee;
+    [Write, Description("The access rights granted to the account. Only 'SendAs' is supported.")] String AccessRights;
```

Comment:
If only SendAs is supported, shouldn't we hardcode that in the resource? Doesn't make sense to request input from the user, but require a specific value.

Reply:
Right, done that.